api/src/main/webapp/WEB-INF/cron_default.yaml

```diff
@@ -80,4 +80,10 @@ cron:
   schedule: every day 19:00
   timezone: America/Chicago
   target: api
+- description: >
+    Compile a snapshot of workbench data for reporting and upload it to  BigQuery.
+  url: /v1/cron/expireNonCompliantUsers
```

Comment:
Could we generalize the name of this method? It's not just expiring users, it is doing the more general task of synchronizing all user data access levels to be aligned with the system definitions (note: this behavior is desirable, and addresses a long-standing issue https://precisionmedicineinitiative.atlassian.net/browse/RW-2352). e.g. something like `synchronizeUserAccess`

Under normal operations, this cron would only make changes according to access expiration, but this would be the cron to invoke manually going forward if we changed access module settings, for example.